api/swagger.yaml

```diff
@@ -434,6 +434,9 @@ definitions:
           Mode:
             description: "The permission mode for the tmpfs mount in an integer."
             type: "integer"
+          Options:
+            description: "The list of options to be passed to the tmpfs mount in a string."
+            type: "string"
```

Comment:
ISTR there was some discussion whether this should be a `string`, a `[]string`  (or a `map[string]string`, but I think we didn't want a map), did we ever reach full consensus on that?